lambdas/FetchJobsDataFiltered.py

```diff
@@ -0,0 +1,138 @@
+import psycopg2
+import psycopg2.extras
+import json
+import os
+import logging
+from typing import Dict, List, Any
+import datetime
+
+try:
+    # for local development, load environment variables from .env file
+    import dotenv
+    dotenv.load_dotenv()
+except ImportError:
+    logging.warning('dotenv not available, assuming running in AWS environment')
+    
+# Constants for environment variables
+DB_HOST = os.environ['DB_HOST']
+DB_PORT = os.environ['DB_PORT']
+DB_NAME = os.environ['DB_NAME']
+DB_USER = os.environ['DB_USER']
+DB_PASSWORD = os.environ['DB_PASSWORD']
+
+# Province mapping for Canadian provinces in a standardized format
+PROVINCE_MAPPING = {
+    "ab": "alberta",
+    "bc": "britishcolumbia",
+    "mb": "manitoba",
+    "nb": "newbrunswick",
+    "nl": "newfoundlandandlabrador",
+    "ns": "novascotia",
+    "on": "ontario",
+    "pe": "princeedwardisland",
+    "qc": "quebec",
+    "sk": "saskatchewan",
+    "nt": "northwestterritories",
+    "nu": "nunavut",
+    "yt": "yukon"
+}
```

Comment:
I don't know how to share code between different functions in lambda, but there might be other functions that use these constants in the future. We need to do some investigation. 

I'm pretty sure there is a way to write a full blown repository for backend service using AWS lambda. not just separate files with separate code.